ui/admin/tests/acceptance/app-tokens/read-test.js

```diff
@@ -0,0 +1,223 @@
+
+    // Check for Details tab
+    assert.dom(commonSelectors.HREF(urls.appToken)).exists();
+
+    // Check for Permissions tab
+    assert.dom(commonSelectors.HREF(urls.appTokenPermissions)).exists();
+  });
+
+  test('can navigate between Details and Permissions tabs', async function (assert) {
+    setRunOptions({
+      rules: {
+        'color-contrast': {
+          enabled: false,
+        },
+      },
+    });
+
+    await visit(urls.appToken);
+    assert.strictEqual(currentURL(), urls.appToken);
+
+    // Navigate to Permissions tab
+    await click(commonSelectors.HREF(urls.appTokenPermissions));
+    assert.strictEqual(currentURL(), urls.appTokenPermissions);
+
+    // Navigate back to Details tab
+    await click(commonSelectors.HREF(urls.appToken));
+    assert.strictEqual(currentURL(), urls.appToken);
+  });
+
+  test('Details tab displays app token form', async function (assert) {
+    setRunOptions({
+      rules: {
+        'color-contrast': {
+          enabled: false,
+        },
+      },
+    });
+
+    await visit(urls.appToken);
+
+    // Check that the form component is rendered
+    assert.dom('.rose-form').exists();
+
+    // Check for form field labels
+    assert.dom('.rose-form').containsText('Name');
+    assert.dom('.rose-form').containsText('Description');
+
+    // Check for section titles
+    assert.dom('.rose-form').containsText('About this token');
+    assert.dom('.rose-form').containsText('Time to live');
+    assert.dom('.rose-form').containsText('Time to stale');
+
+    // Check for disabled inputs (HDS form fields render as disabled, not readonly)
+    assert.dom('input[disabled]').exists('Should have disabled fields');
+
+    // Check for description lists (About, TTL, TTS sections)
+    assert.dom('.description-list').exists({ count: 3 });
+
+    // Check for status badge
+    assert.dom('.hds-badge').exists();
+  });
+
+  test('app token page displays status badge', async function (assert) {
+    setRunOptions({
+      rules: {
+        'color-contrast': {
+          enabled: false,
+        },
+      },
+    });
+
+    await visit(urls.appToken);
+
+    // Check for status badge
+    assert.dom('.hds-badge').exists();
+    assert.dom('.hds-badge').containsText('Active');
+  });
+
+  test('Permissions tab renders placeholder', async function (assert) {
+    setRunOptions({
+      rules: {
+        'color-contrast': {
+          enabled: false,
+        },
+      },
+    });
+
+    await visit(urls.appTokenPermissions);
+
+    // Permissions tab should have a form (even if empty for now)
+    assert.dom('.rose-form').exists();
+  });
+
+  test('page title is set to app token display name', async function (assert) {
+    setRunOptions({
+      rules: {
+        'color-contrast': {
+          enabled: false,
+        },
+      },
+    });
+
+    await visit(urls.appToken);
+
+    // Check that page title includes token name
+    assert.dom('.hds-page-header__title').exists();
+  });
```

Comment:
This isn't asserting what the test name says. Could we remove this test entirely? Right now the title should just be "App Tokens" and we could include that assert in the first test maybe?